device-types/Cisco/C9800-L-F.yaml

```diff
@@ -0,0 +1,36 @@
+---
+manufacturer: Cisco
+model: Catalyst 9800-L-F
+slug: cisco-c9800-l-f
+part_number: C9800-L-F
+u_height: 1
+is_full_depth: false
+comments: '[Cisco Catalyst 9800-L Wireless Controller Data Sheet](https://www.cisco.com/c/en/us/products/collateral/wireless/catalyst-9800-series-wireless-controllers/datasheet-c78-742434.html)'
+console-ports:
+  - name: Con
+    type: rj-45
+  - name: usb0
+    type: usb-micro-b
+interfaces:
+  - name: Service Port
+    type: 1000base-t
+    mgmt_only: true
+  - name: Redundancy Port
```

Comment:
In the configuration, what is the actual ports named? Is it actually called `Redundancy Port`?

Reply:
Sorry, don't have access to a config at the moment, will try and find out the real name of this interface.

Reply:
Please let me know when you have found this information